### PR TITLE
added pdf-tools and workgroups2

### PR DESCRIPTION
**Pull request by deyaaeldeen**
btw I have created a new category for PDF, since I did not find a related category for pdf-tools mode. I put it right after latex.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -6,7 +6,7 @@ Most of the following packages are available in [[https://github.com/milkypostma
 
 #+BEGIN_QUOTE
 *Table of Contents*
-- [[#awesome-emacs][Awesome Emacs]]
+- [[#awesome-emacs-[[httpsgithubcomsindresorhusawesome][httpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvg]]][Awesome Emacs [[https://github.com/sindresorhus/awesome][https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg]]]]
```
Comment (kuanyui) [inline README.org]: What on earth is this? Explain it.

Comment (deyaaeldeen) [inline README.org]: This happens when I call awesome-emacs-gen-toc as instructed in the Contributing section. I have no idea what it is.

Comment (kuanyui) [inline README.org]: I'm Sorry..., that sounds like a bug of gen-toc. I'll take a look of it.

Comment (kuanyui) [inline README.org]: I got it. Someone append a svg icon link just after the title (without newline) and `gen-toc` treat it as normal string.

And...Why delete PR....

Comment (deyaaeldeen) [inline README.org]: I thought it is my fault somehow. Anyway glad it is fixed.

**Pull request closed without merging**